### PR TITLE
[april fools] makes felinids more realistic

### PR DESCRIPTION
**Pull request by scriptis**
## About The Pull Request

makes felinids more realistic

this has been improved since february. i don't know how don't ask me
## Why It's Good For The Game

makes felinids more realistic
## Changelog
:cl:
add: felinids are now more realistic
fix: felinids can no longer strip people (gross)
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Please stop voring people :(

**Comment by scriptis** [comment]
> Please stop voring people :(

im unstoppable

**Comment by Mooshimi** [comment]
no thanks

File: code/modules/mob/living/carbon/alien/adult/alien_powers.dm [hunk 1]
```diff
@@ -162,9 +162,9 @@ Doesn't work on other aliens/AI.*/
 
 /datum/action/cooldown/alien/transfer/Activate(atom/target)
 	var/mob/living/carbon/carbon_owner = owner
-	var/list/mob/living/carbon/aliens_around = list()
-	for(var/mob/living/carbon/alien in view(owner))
-		if(alien.getPlasma() == -1 || alien == owner)
+	var/list/mob/living/carbon/alien/aliens_around = list()
+	for(var/mob/living/carbon/alien/alien in view(owner))
+		if(alien == owner)
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/alien/adult/alien_powers.dm:167]: why is this only looking for aliens? can't humans have alien plasma organs implanted in them?

**Comment by scriptis** [comment]
i regret everything. i regret everything i've ever done. why have i done this the results are far worse than i could have possibly imagined oranges was wrong

**Comment by san7890** [comment]
you toyed with powers beyond your comprehension. i hope you now understand the meaning of the word "suffering".

File: code/modules/mob/living/carbon/alien/adult/adult.dm [hunk 2]
```diff
@@ -111,7 +111,7 @@ GLOBAL_LIST_INIT(strippable_alien_humanoid_items, create_strippable_list(list(
 	return ..()
 
 /// Returns FALSE if we're not allowed to eat it, true otherwise
-/mob/living/carbon/alien/adult/proc/can_consume(atom/movable/poor_soul)
+/mob/living/carbon/proc/can_consume(atom/movable/poor_soul)
```
Comment (LemonInTheDark) [inline code/modules/mob/living/carbon/alien/adult/adult.dm:114]: And you could have it all, my empire of dirt, 
I will let you down, I will make you hurt.

I wear this crown of thorns, upon my liar's chair,
Full of broken thoughts, I can not repair.

Beneath the stains of time, the feelings disappear,
You are someone else, I am still right here.

What have I become, my sweetest friend,
Everyone I know, goes away in the end.

File: code/modules/mob/living/carbon/alien/adult/alien_powers.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/mob/living/carbon/alien/adult/alien_powers.dm:167]: No. You may not vore your fellow crew memebers.

**Comment by Mooshimi** [comment]
> i regret everything. i regret everything i've ever done. why have i done this the results are far worse than i could have possibly imagined oranges was wrong
https://github.com/tgstation/tgstation/pull/74420#issuecomment-1492829224

**Comment by Moonlit2000** [comment]
Will we be getting a followup to this this year?

**Comment by scriptis** [comment]
yes

**Comment by LemonInTheDark** [comment]
no

**Pull request closed without merging**